sqlglot/expressions.py

```diff
@@ -818,7 +818,10 @@ class SetItem(Expression):
         "expressions": False,
         "kind": False,
         "collate": False,  # MySQL SET NAMES statement
-        "global": False,
+        "global_": False,
```

Comment:
just call it global since it's a dictionary, when you set it you can use a dict, this is how it's done for "with"

Reply:
I've done a text search for `with=` and `with =` to find how the reserved word has been used as a variable name and can't find it.

It's not a clash with a reserved word here, but it is in parser.py (about line 3861) where the keys in the dict defined here are expanded to be named arguments to the expression constructor.

Reply:
you can’t use kwargs, you need to use a dict

Reply:
yeah that would do it